### ISSUE TITLE
IP Blocked for API Key

### ISSUE THREAD

**Issue by Chap92** [delta-exchange/python-rest-client#47]
Created a bot for trading using the API. The bot initiates a trade after checking the price and then, keeps-on checking the price every 20 seconds until the correct conditions are met. This is way below the rate limit, However, after a few hours (sometimes minutes), I get this message:
` {'code': 'ip_blocked_for_api_key', 'context': {'client_ip': '2687:sgw:2878:229:1'}}`
IP address shown is fake; just for demonstration purposes. Why is this happening? The bot is hosted on a VPS server different from the location I'm living in. Is that it? Is the Delta server blacklisting me because my trading server and the computer I access my account from are in 2 different regions? OR something else? 

Anyone eles faced this problem?
P.S I did whitelist both the IPv4 and IPv6 addresses.

**Comment by sanchayjain28** [comment]
Yes I faced this same problem but after whitelisting IPV6 it is working

**Comment by Chap92** [comment]
That was the first thing I tried. It fixes the problem on my local machine, but  on the VPS server it doesn't work. On the VPS server, after the initial calls to the API, the IPv6 address gets blacklisted. What's frustrating is that no dev on this project is addressing these issues. I even created a support ticket. They can simply look at the logs and tell why it is getting blacklisted.
